lib/l10n_utils/__init__.py

```diff
@@ -52,10 +53,13 @@ def render(request, template, context=None, **kwargs):
     if isinstance(template, list):
         template = template[0]
 
+    if ftl_files:
+        context['ftl_files'] = ftl_files
```

Comment:
I'd create something like a `FluentLocalization` here instead. That'll allow you to cache bundles, which, sadly, are quite expensive to create.

I've started to hack on a branch for that on https://github.com/Pike/python-fluent/pull/1.